Dzianis_Zavoiskikh/11.Ansible.start/dz_hosts.yaml

```diff
@@ -0,0 +1,26 @@
+ec_dz:  

```

Comment:
Do not duplicate code, please use group variable (see section with credentials )